src/main/java/frc/robot/commands/ShooterEject.java

```diff
@@ -0,0 +1,68 @@
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.Constants.AimModes;
+import frc.robot.subsystems.CDSSubsystem;
+import frc.robot.subsystems.ShooterSubsystem;
+
+public class ShooterEject extends CommandBase {
+  private ShooterSubsystem m_ShooterSubsystem;
+  private CDSSubsystem m_CDSSubsystem;
+  private int i;
+  private AimModes tempMode;
+
+  /** Creates a new ShooterPressed. */
+  public ShooterEject(ShooterSubsystem shooterSubsystem, CDSSubsystem cdsSubsystem) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(shooterSubsystem);
+    addRequirements(cdsSubsystem);
+    m_ShooterSubsystem = shooterSubsystem;
+    m_CDSSubsystem = cdsSubsystem;
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    tempMode = m_ShooterSubsystem.getAimMode();
+    i = 0;
+    m_ShooterSubsystem.setAimMode(Constants.AimModes.EJECT);
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    m_ShooterSubsystem.prime();
+    if (m_ShooterSubsystem.wheelReady()) {
+      i++;
+      m_CDSSubsystem.CDSBeltToggle(false);
+      m_ShooterSubsystem.runCargo(0.65);
```

Comment:
This should be a constant.  It is used in multiple commands.  Same with -0.4 below

Reply:
I don't know how else I would set this, if not periodically?

I'll turn the speeds into constants

Reply:
once you set the speed of a motor, doesn't it stay until you set it again? 

If this is another hardware call, then it would be good to call it once, if the speed is supposed to stay the same until it is changed.

Reply:
The speed is supposed to stay the same until changed, but where do you expect me to change the speed if not in execute()?

Reply:
I am suggesting that you change it once in execute.  

https://docs.wpilib.org/en/stable/docs/software/commandbased/commands.html#execution

Execute is called every 20ms.  So, call it the first time to set it, then not after i>0